testsuite/testsuite-container/src/test/java/org/wildfly/swarm/container/test/DeploymentFailureTest.java

```diff
@@ -0,0 +1,31 @@
+package org.wildfly.swarm.container.test;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.junit.Test;
+import org.wildfly.swarm.Swarm;
+import org.wildfly.swarm.container.DeploymentException;
+import org.wildfly.swarm.spi.api.JARArchive;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.fest.assertions.Fail.fail;
+
+public class DeploymentFailureTest {
+
+  @Test
+  public void testDeploymentFailure() throws Exception {
+      Swarm swarm = new Swarm();
```

Comment:
Due to VFS TempFileProvider storing temporal file paths in static section, it doesn't goes well under same test class (TempFileProvider won't be unloaded). Separating in two different test classes does the trick.